src/components/pages/Profile/RenderCustomerProfile.js

```diff
@@ -92,37 +89,35 @@ const RenderCustomerProfile = ({ userInfo }) => {
     setIsModalVisible1(false);
   };
 
-  const handleSave = formData => {
-    const newPets = petData.map(newpet => {
-      if (newpet.id === formData.id) {
-        return {
-          id: formData.id,
-          pet_name: formData.pet_name,
-          color: formData.color,
-          date_of_birth: formData.date_of_birth,
-          image_url: formData.image_url,
-          phone_number: formData.phone_number,
-        };
-      } else {
-        return newpet;
-      }
-    });
-    setPetData(newPets);
-  };
-
-  const handleDelete = id => {
-    console.log('ID', id);
-    const newPets = petData.filter(pet => pet.id !== id);
-    setPetData(newPets);
-  };
-
-  const createPet = data => {
-    const newId = petData[petData.length - 1].id + 1;
-    const newPet = { id: newId, ...data };
-    setPetData([...petData, newPet]);
-  };
-
-  console.log('PETDATA', petData);
+  // const handleSave = formData => {
+  //   const newPets = petData.map(newpet => {
+  //     if (newpet.id === formData.id) {
+  //       return {
+  //         id: formData.id,
+  //         pet_name: formData.pet_name,
+  //         color: formData.color,
+  //         date_of_birth: formData.date_of_birth,
+  //         image_url: formData.image_url,
+  //         phone_number: formData.phone_number,
+  //       };
+  //     } else {
+  //       return newpet;
+  //     }
+  //   });
+  //   setPetData(newPets);
+  // };
+
+  // const handleDelete = id => {
+  //   console.log('ID', id);
+  //   const newPets = petData.filter(pet => pet.id !== id);
+  //   setPetData(newPets);
+  // };
+
+  // const createPet = data => {
+  //   const newId = petData[petData.length - 1].id + 1;
+  //   const newPet = { id: newId, ...data };
+  //   setPetData([...petData, newPet]);
+  // };
```

Comment:
there is a lot of code commented out here -- not sure if we should merge this particular file to main if it's not ready yet